File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/toolbar/ToolbarInterceptorCallback.java
package com.raxdenstudios.square.interceptor.commons.toolbar;

import android.os.Bundle;

import com.raxdenstudios.square.interceptor.InterceptorCallback;

import androidx.appcompat.widget.Toolbar;

/**
 * Created by agomez on 21/05/2015.
 */
public interface ToolbarInterceptorCallback extends InterceptorCallback {

    Toolbar onCreateToolbarView(Bundle savedInstanceState);

    void onToolbarViewCreated(Toolbar toolbar);

}

File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/toolbar/ToolbarInterceptorCallback.kt
package com.raxdenstudios.square.interceptor.commons.toolbar

import android.os.Bundle
import android.support.v7.widget.Toolbar

import com.raxdenstudios.square.interceptor.InterceptorCallback

/**
 * Created by agomez on 21/05/2015.
 */
interface ToolbarInterceptorCallback : InterceptorCallback {

    fun onCreateToolbarView(savedInstanceState: Bundle?): Toolbar?

    fun onToolbarViewCreated(toolbar: Toolbar)

}
